### DISCUSSION TITLE
[HELP] I downloaded the latest version of fooocus and I don't know how to install it

### DISCUSSION THREAD

**Discussion by elyzionz** [lllyasviel/Fooocus#3129] [Q&A] [answered]
Hello, I am new to this, I downloaded version 2.4.3 of fooocus .bat I don't see any bat file to run and install it, can someone explain to me? thanks in advance, I use windows

**Comment by monstamash77** [comment]
You can maybe check out Stability Matrix: https://github.com/LykosAI/StabilityMatrix

Very user friendly with lots of cool features, including Fooocus one click install

**Comment by mashb1t** [comment, marked as answer]
If you want to install it the official way please download the zip file at https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z ([see here](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#windows)), where the .bat files and all necessary python packages are already included.

**Comment by Vanessa88Love** [comment]
I managed to download the zip files and extract them, but the programme still doesn't start like it should. Could somebody tell me what there is lacking? And can I fix it in some way? When running run_realistic.bat Command Prompt tells requirement are not reached, but I don't understand what is lacking. Please see the screenshot. 
My computer has as processor Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz   2.59 GHz and RAM 32,0 GB (31,8 GB usable), plus is has Nvidia graphic card - I didn't find the info again, but I think I saw Nvidia 20?0 version of it. 
![donotmeetrequirements](https://github.com/user-attachments/assets/d485b391-04ff-4fad-a9eb-12421083cff9)
![Failed](https://github.com/user-attachments/assets/67dfb03c-7f7d-4753-9dfb-670f14c857d7)

**Reply by mashb1t** [reply]
@Vanessa88Love Please disable VPN / Proxies / DNS blockers / Browser extensions and check your firewll, it seems like your PC can't download the package due to being blocked.

**Reply by Vanessa88Love** [reply]
Disabling the VPN helped. Thanks a lot! Now it's working! :-D

**Reply by MartyC22** [reply]
Everytime I try to download it I get the error message, "File wasn't available on site."

**Reply by mashb1t** [reply]
@MartyC22 for which file? Please provide more details.

**Comment by MartyC22** [comment]
I've tried both of these and get the same error message. I've been
downloading other files all day and haven't had any problems.

*>>> Click here to download <<<
<https://github.com/lllyasviel/Fooocus/releases/download/v2.5.0/Fooocus_win64_2-5-0.7z>*

https://github.com/LykosAI/StabilityMatrix

**Reply by mashb1t** [reply]
@MartyC22 Same for you, please disable VPN / Proxies / DNS blockers / Browser extensions and check your firewall.
Can confirm the 7z link is working perfectly fine, but maybe Github downloads are somehow blocked for you.

**Comment by lowiq00** [comment]
Helpppppppppppp! i tried more than 10 time downloading fooocus zip but when the download reaches 85% to 90% it stop and resume does not work either, tried on chrome browser & idm, tried over and over no luck and give up.

**Reply by mashb1t** [reply]
@lowiq00 https://github.com/lllyasviel/Fooocus/discussions/3129#discussioncomment-10226709 i know low IQ (lol) but did this help?

**Comment by nabsers1** [comment]
Hi I am facing the same issue. I have disabled all VPNs etc, but still the download is disrupted at 1.1/1.9GB every time

**Reply by Rogala** [reply]
[Try a file downloader that can resume downloading in case of interruption or failure.](https://portableapps.com/apps/internet/free-download-manager-portable)
